### PR TITLE
Fix 2/3 of HLT test breakages caused by PR #9216

### PR DESCRIPTION
**Pull request by harmonicoscillator**
Add proper hasattr guards to prevent problems with HLT tests reported in PR #9216

### CODE REVIEW

**Comment by richard-cms** [comment]
@mulhearn @Martin-Grunewald

**Comment by richard-cms** [comment]
This PR is necessary but not sufficient to fix the HLT test failures in the HI workflows. We also need to update the HLT workflows to include the caloConfig object (the pp workflow that is broken also needs this), but I think @mulhearn or @jimbrooke know better where that should go.

File: L1Trigger/L1TCommon/python/customsPostLS1.py [hunk 1]
```diff
@@ -101,15 +101,18 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
...
-    process.caloStage1Params.regionPUSType = cms.string("zeroWall")
-    process.caloConfig.fwVersionLayer2 = cms.uint32(1)
+    if hasattr(process, 'RCTConfigProducers'):
+        process.RCTConfigProducers.eicIsolationThreshold = cms.uint32(7)
+        process.RCTConfigProducers.hOeCut = cms.double(999)
+        process.RCTConfigProducers.eMinForHoECut = cms.double(999)
+        process.RCTConfigProducers.eMaxForHoECut = cms.double(999)
+        process.RCTConfigProducers.hMinForHoECut = cms.double(999)
+        process.RCTConfigProducers.eMinForFGCut = cms.double(999)
+    if hassattr(process, 'caloStage1Params'):
```
Comment (Martin-Grunewald) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: Should be hasattr (not hassattr)

File: L1Trigger/L1TCommon/python/customsPostLS1.py [hunk 2]
```diff
@@ -101,15 +101,18 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
...
+        process.caloStage1Params.jetSeedThreshold = cms.double(0.)
+        process.caloStage1Params.regionPUSType = cms.string("zeroWall")
+    if hassattr(process, 'caloConfig'):
```
Comment (Martin-Grunewald) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: Should be hasattr (not hassattr)

**Comment by davidlange6** [comment]
I'm making a PR based on this one with a fix that will at least run without crashing. Closing this one

**Pull request closed without merging**